### PR TITLE
ADEN-10661 Removing adengine experimental variables

### PR DESCRIPTION
**Pull request by dmnsobczak**

### CODE REVIEW

File: extensions/wikia/AdEngine3/AdEngine3.class.php [hunk 1]
```diff
@@ -115,8 +115,6 @@ public static function getContext( Title $title = null ) {
 			return [
 				'bidders' => [],
 				'opts' => array_filter([
-					'adEngineVersion' => $wg->AdEngineVersion,
```
Comment (Loczi94) [inline extensions/wikia/AdEngine3/AdEngine3.class.php:118]: Correct me if I'm wrong but I think we use it for tracking ad-engine version.

File: includes/wikia/DefaultSettings.php [hunk 2]
```diff
@@ -1008,20 +1008,6 @@
  */
 $wgAdDriverTrackState = false;
 
-/**
- * Version of AdEngine bundle
- * @see extensions/fandom/AdEngine
- * @var string $wgAdEngineVersion
- */
-$wgAdEngineVersion = null;
```
Comment (Loczi94) [inline includes/wikia/DefaultSettings.php:1016]: ditto

File: extensions/wikia/AdEngine3/AdEngine3.class.php (hunk 1 above)
Comment (dmnsobczak) [reply extensions/wikia/AdEngine3/AdEngine3.class.php:118]: Disagree. There are two usages of this variable in mobile-wiki. First for reporting, but this value comes from `window.ads` and it is populated by ad-engine itself:
https://github.com/Wikia/mobile-wiki/blob/78505e69dfd462a39aec4f4f932b68a473bf9468/app/modules/ads/core/stable-ads.js#L109
Second is used in adengine-experimental feature and this part of code will be unused:
https://github.com/Wikia/mobile-wiki/blob/e82bd3b86e4d6ee03e4bf937e50ef2f75fe1b02a/app/modules/ads/core/experimental-ads.js#L33
Please review.

**Comment by dmnsobczak** [comment]
One additional comment: I decided to left AdEngine Experimental code untouched in mobile-wiki repo. Szymon and Bielik did a bit of refactor during developing that, so reversing those commits is like going back and cutting out Experimental feature will require QA against regression.
Can you agree with leaving this code as it is and letting it die (being sunset) naturally during New Mobile Experience at the end of the year?

**Review by Loczi94 — APPROVED** [review]

**Pull request merged**